travis/licenses_golden/licenses_third_party

```diff
@@ -34,6 +34,214 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 
+====================================================================================================
+ORIGIN: ../../../third_party/googletest/googlemock/scripts/generator/LICENSE
+TYPE: LicenseType.apache
+----------------------------------------------------------------------------------------------------
+Apache License
```

Comment:
since you have a new license here please make sure you regenerate the actual LICENSE file.

On the other hand, I assume we don't actually _ship_ googletest? In which case, you should probably exclude this entire subdirectory from the license script.